cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1485,6 +1503,31 @@
   </property>
 
   <property>
+    <name>metrics.messaging.fetcher.persist.threshold</name>
+    <value>500</value>
+    <description>
+      Threshold of fetching messages from messaging fetcher and processing
+      metrics messages before persisting the processed metrics to metrics table
+    </description>
+  </property>
+  
+  <property>
+    <name>metrics.messaging.meta.table</name>
+    <value>metrics.messaging.meta</value>
+    <description>
+      Name of the messaging metrics meta table
+    </description>
+  </property>
+
+  <property>
+    <name>metrics.messaging.partition.size</name>
+    <value>10</value>
+    <description>
+      Number of partitions for messaging metrics topic
```

Comment:
yep, though some just need a restart on the master process, while others need a restart on the master services running on yarn (master services are not restart in HA mode).